pkg/epinio/detail/applications.vue

```diff
@@ -125,73 +194,51 @@ export default Vue.extend<Data, any, any, any>({
               </thead>
               <tr>
                 <td>{{ t('tableHeaders.memory') }}</td>
-                <td>{{ value.instanceMemory.min }}</td>
-                <td>{{ value.instanceMemory.max }}</td>
-                <td>{{ value.instanceMemory.avg }}</td>
+                <td>{{ resourcesUssage.memoryBytes.toFixed(2) }} MiB</td>
```

Comment:
The memory and cpu stats were already calculated in the application model. Can't remember if that was an older change or not, but we should probably use that version